qa/tasks/cephfs/test_volumes.py

```diff
@@ -512,6 +589,37 @@ def test_subvolume_snapshot_in_group(self):
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
 
+    def test_subvolume_snapshot_ls(self):
+        # tests the 'fs subvolume snapshot ls' command
+
+        snapshots = []
+
+        # create subvolume
+        subvolume = self._generate_random_subvolume_name()
+        self._fs_cmd("subvolume", "create", self.volname, subvolume)
+
+        # create subvolume snapshots
```

Comment:
No need to repeat the commands three times. Do this in a loop as mentioned above.